js/script.js

```diff
@@ -18,6 +18,7 @@ const scoreboard = {
   computerScore: 0
 };
 
+const choices = ['rock', 'paper', 'scissors'];
```

Comment:
You forgot to define all possible options which are needed to get the computers choice